tests/integration/test_writes/test_writes.py

```diff
@@ -1409,3 +1399,52 @@ def test_delete_threshold() -> None:
     assert len(iceberg_table.scan(row_filter=Not(delete_condition)).to_arrow()) == lower_before
     iceberg_table.delete(delete_condition)
     assert len(iceberg_table.scan().to_arrow()) == lower_before
+
+
+@pytest.mark.integration
+def test_writes_jasper(session_catalog: Catalog) -> None:
```

Comment:
nit: should we rename this to something a bit more descriptive? like: `test_overwrite_with_partition_evolution` ?

Reply:
@sungwy Great catch! That was a place-holder :)
```suggestion
def test_rewrite_manifest_after_partition_evolution(session_catalog: Catalog) -> None:
```